cmd/kubernetes-history-inspector/main.go

```diff
@@ -41,63 +42,64 @@ import (
 	"github.com/GoogleCloudPlatform/khi/pkg/source/gcp/api/accesstoken"
 	"github.com/GoogleCloudPlatform/khi/pkg/source/gcp/api/quotaproject"
 
-	"cloud.google.com/go/profiler"
+	"log/slog"
 )
 
-func displayStartMessage(host string, port int) {
-	var (
-		bold  = "\033[1m"
-		green = "\033[32m"
-		cyan  = "\033[36m"
-		reset = "\033[0m"
-	)
-	if parameters.Debug.NoColor != nil && *parameters.Debug.NoColor {
-		bold = ""
-		green = ""
-		cyan = ""
-		reset = ""
-	}
-	hostInHintText := host
-	if host == "0.0.0.0" || host == "127.0.0.1" {
-		hostInHintText = "localhost"
-	}
-	fmt.Printf(`%[1]s%[2]s%[3]s Starting KHI server with listening %[4]s:%[5]d%[1]s`, reset, bold, green, host, port)
-	if hostInHintText == "localhost" {
-		fmt.Printf(`
-%[4]s%[2]sFor Cloud Shell users:
-	Click this address >> %[3]shttp://%[5]s:%[6]d%[1]s%[2]s%[4]s << Click this address
-
-%[1]s%[4]s(For users of the other environments: Access %[3]shttp://%[5]s:%[6]d%[1]s%[4]s with your browser. Consider SSH port-forwarding when you run KHI over SSH.)
-%[1]s`, reset, bold, green, cyan, hostInHintText, port)
-	}
-}
-
-var taskSetRegistrer []inspection.PrepareInspectionServerFunc = make([]inspection.PrepareInspectionServerFunc, 0)
+var taskSetRegistrer []inspection.PrepareInspectionServerFunc
```

Comment:
```suggestion
var taskSetRegistrer []inspection.PrepareInspectionServerFunc = make([]inspection.PrepareInspectionServerFunc, 0)
```
The reason `taskSetRegisterer` is initialized with an empty array is actually so that multiple files in the `cmd` folder have `init()` and can easily extend the list of tasks.

For example, a user who wants to add functionality for their internal purpose can add a file like `private.go` to `cmd/kubernetes-history-inspector/`, add a task to `init()` to add some set of tasks for their internal purpose, and customize it for internal use by ignoring only `private.go` with `.gitignore`.

I know it's impossible to get this purpose just from this code, but this is why we initialized the array with empty array and append tasks on `init()` instead of assigning a value to the `taskRegisterer`.